common.js

```diff
@@ -138,6 +138,13 @@ var vcwg = {
       title: 'InterPlanetary File System (IPFS)',
       href: 'https://en.wikipedia.org/wiki/InterPlanetary_File_System',
       publisher: 'Wikipedia'
+    },
+    'JSON-SCHEMA-2019': {
```

Comment:
One niggle occurs to me. That draft was issued in March 2018 and expired in September 2018, and we're less than halfway through 2019.  Perhaps the reference `JSON-SCHEMA-2019` should instead be `JSON-SCHEMA-2018` (as there may yet be a 2019-dated revision)?  (I don't know whether these are strictly local identifiers, or if there's some desired persistence/consistency/reuse across W3 or other docs.)

Reply:
The identifiers in this file are local but often get migrated to the specref database.  I agree with considering a JSON-SCHEMA-2018.

Reply:
Fixed in 7175fad7f53eaf29ae26051b8bb701e984649e4e.